integration_test/tests/util.rs

```diff
@@ -0,0 +1,86 @@
+// SPDX-License-Identifier: CC0-1.0
+
+//! Tests for methods found under the `== Util ==` section of the API docs.
+
+#![allow(non_snake_case)] // Test names intentionally use double underscore.
+
+use bitcoin::{PublicKey, PrivateKey};
+use integration_test::{Node, NodeExt as _, Wallet};
+use node::vtype::*;
+use node::mtype;
+
+#[test]
+fn util__create_multisig__modelled() {
+    let nrequired = 2;
+
+    // Use two valid, deterministic public keys from the pubkey_sort test vectors.
+    let pubkey1 = "02ff12471208c14bd580709cb2358d98975247d8765f92bc25eab3b2763ed605f8"
+        .parse::<PublicKey>()
+        .unwrap();
+    let pubkey2 = "02fe6f0a5a297eb38c391581c4413e084773ea23954d93f7753db7dc0adc188b2f"
+        .parse::<PublicKey>()
+        .unwrap();
+
+    let node = Node::with_wallet(Wallet::Default, &[]);
+    let json: CreateMultisig = node
+        .client
+        .create_multisig(nrequired, vec![pubkey1, pubkey2])
+        .expect("createmultisig");
+    let res: Result<mtype::CreateMultisig, _> = json.into_model();
```

Comment:
```suggestion
    let res: Result<mtype::CreateMultisig, CreateMultisigError> = json.into_model();
```

Use `_` if the error is a rust-bitcoin error and use the explicit error if it is from `types` - that proves we re-exported the error correctly.